etc/samples/flows_generic.sample.yaml

```diff
@@ -0,0 +1,28 @@
+# This schema is not used stand-alone, it is a sub-section of "tendrl_definitions_<sds_nvr>.yaml" | "tendrl_definitions_generic.sample.yaml
+
+---
+  flows:
+    CreateCephCluster:
+      enabled: True
+      tendrl_context: "A map of Tendrl flow level context items like cluster id,tendrl id etc"
+      uuid: "todo"
+      description: "Create Ceph Cluster with given context/params"
+      atoms: ["lots_of_atoms_like_install_rpms_bookeeping_etc"]
+      pre_run : ["tendrl.node_agent.atoms.<atom_name>.<boolean_function>1"...]
+                #boolean returning pre/post conditions function list
+      post_run: ["tendrl.node_agent.atoms.<atom_name>.<boolean_function>1"...]
+      run: "tendrl.node_agent.flows.create_ceph_cluster"
+      type: Create
+      outputs:
+      - "A map of type {object.attr1 : {required: True}, object
+     .attr2:{required: False}}}} that this will be providing (or could
+     provide) to others, used to correlate and associate the thing/s this flow
+      produces, if it produces anything at all."
+      inputs:
+      - An immutable {object1.attr1 : {value: "cluster1"}, object
```

Comment:
Flow and atoms are separate entities, please validate them separately. 

The reason we have inputs and outputs for both flows and atoms is so that we can have flow level data and atom level data.